core/settings/qPrefPrivate.h

```diff
@@ -14,213 +14,181 @@ class qPrefPrivate : public QObject {
     Q_OBJECT
```

Comment:
What's the point of making this class a `QObject`? I don't see any signals or the like?

Reply:
I had problems with QSettings if the Q_OBJECT was not present.

Reply:
I had this in other classes as well - over time we had a number of classes that ended up all being QObjects because things broke otherwise and I didn't understand why.